exporter.py

```diff
@@ -220,15 +228,18 @@ def generate_metrics(self):
                         paramDesc = paramsetDescription.get(key)
                         paramType = paramDesc.get('TYPE')
                         if paramType in ['FLOAT', 'INTEGER', 'BOOL']:
+                            logging.debug("Found {}: desc: {} key: {}".format(paramType, paramDesc, paramset.get(key)))
                             self.process_single_value(devAddress, devType, devParentAddress, devParentType, paramType, key, paramset.get(key))
                         elif paramType == 'ENUM':
                             logging.debug("Found {}: desc: {} key: {}".format(paramType, paramDesc, paramset.get(key)))
                             self.process_enum(devAddress, devType, devParentAddress, devParentType,
                                               paramType, key, paramset.get(key), paramDesc.get('VALUE_LIST'))
+                        elif paramType in ['ACTION', 'STRING']:
+                            logging.debug("Ignoring unsupported paramType {}, desc: {}, key: {}".format(paramType, paramDesc, paramset.get(key)))
```

Comment:
Why do you explicitly want to log these paramTypes when they'd be logged in `else` anyway?